com.google.AndroidStudio.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.google.AndroidStudio.desktop</id>
+	<metadata_license>CC-BY-SA-3.0</metadata_license>
+	<project_license>Apache-2.0, BSD-2-Clause</project_license>
```

Comment:
I believe the correct syntax is: `Apache-2.0 AND BSD-2-Clause`.

https://spdx.org/spdx-specification-21-web-version#h.jxpfx0ykyb60

Reply:
Further, I don't actually think that's true for this extra data version, which is subject to the android SDK licence and is redistributible but proprietary.